driver/src/main.rs

```diff
@@ -124,6 +124,15 @@ struct Options {
         parse(try_from_str = duration_millis),
     )]
     gas_station_timeout: Duration,
+
+    /// Time interval in seconds in which price sources should be updated.
+    #[structopt(
+        long,
+        env = "PRICE_SOURCE_UPDATE_INTERVAL",
+        default_value = "1000",
```

Comment:
Let's make the default 5 minutes given the volatility of prices and how cheap these requests are.